### DISCUSSION TITLE
Add These Viacom Kids Channels

### DISCUSSION THREAD

**Discussion by JoveeAcc** [iptv-org/community#427] [Q&A]
These Are The Channels That Need To Be Added To The Iptv Service

Nickelodeon 
NICK 2
Nicktoons West
Nicktoons East
Teennick West
Teenick East
Nick Music
Nick Jr

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel
